### DISCUSSION TITLE
Integrate Mypy into PR checker

### DISCUSSION THREAD

**Discussion by sfc-gh-yihuang2** [camel-ai/camel#829] [General]
Mypy is a static type checker. By enforcing code contributors to think about the data types, it ultimately improves the code quality for a large project. 
https://mypy-lang.org/

**Comment by Wendong-Fan** [comment]
Thanks @sfc-gh-yihuang-2 , we do have mypy check

<img width="831" alt="Screenshot 2024-08-16 at 03 27 57" src="https://github.com/user-attachments/assets/096f3993-83c4-4ffb-a135-cc91c2a09be7">
